### PR TITLE
Deactivated reminder emails

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
fixes https://github.com/mitodl/bootcamp-ecommerce/issues/128

#### What's this PR do?
it removes celery tasks which where responsible of sending emails to user periodically.

#### How should this be manually tested?
the must me no email task schedule

@pdpinch

### CODE REVIEW

**Comment by giocalitri** [comment]
@pdpinch we do not want the reminder emails any more?

**Comment by pdpinch** [comment]
No. Confirmed with multiple parties. 

We don't need them now and if/when the Bootcamp team migrates the admissions application we won't need reminder emails then either.

File: bootcamp/settings.py [hunk 1]
```diff
@@ -426,12 +425,6 @@ def get_var(name, default):
     "CELERY_RESULT_BACKEND", get_var("REDISCLOUD_URL", None)
 )
 CELERY_TIMEZONE = 'UTC'
-CELERY_BEAT_SCHEDULE = {
-    'send-payment-email-reminder-every-24-hrs': {
-        'task': 'mail.tasks.async_send_reminder_payment_emails',
-        'schedule': crontab(minute=0, hour='3')
-    },
```
[the remark this answers is not in the document]
Comment (amir-qayyum-khan) [reply bootcamp/settings.py]: added

**Comment by amir-qayyum-khan** [comment]
@giocalitri should i remove these models

**Comment by giocalitri** [comment]
@amir-qayyum-khan yes: please also check if there is any leftover code introduced by the PR I linked before that is not used any more after we remove the tasks

**Comment by amir-qayyum-khan** [comment]
Done @giocalitri

**Comment by giocalitri** [comment]
@amir-qayyum-khan you should not remove the migrations, but remove the models and add another migration to remove them

**Comment by amir-qayyum-khan** [comment]
Done @giocalitri

**Comment by giocalitri** [comment]
👍

**Pull request merged**